src/pages/pricing/index.tsx

```diff
@@ -45,7 +45,7 @@ const Pricing: FunctionComponent<PricingProps> = ({redirectURL}) => {
       stripeCheckoutRedirect(
         annualPrice.price_id,
         viewer.email,
-        viewer.subscription?.stripe_subscription_id,
+        viewer.subscription?.stripe_customer_id,
```

Comment:
I think this should look at account now, this was pre-account (should hold up for awhile)